pkg/agent/util/dump_linux.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build linux
+
+package util
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/afero"
+	"k8s.io/utils/exec"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/util/iptables"
+	"github.com/vmware-tanzu/antrea/pkg/ovs/ovsctl"
+	"github.com/vmware-tanzu/antrea/pkg/util"
+)
+
+type dumper struct {
+	factory  ofctlClientFactory
```

Comment:
I understand the motivation of being extensible for the possibility that antrea uses multiple bridges in future, but I think it introduces some controversial changes: the interface `ovsctl.ListBridges` is different from all the other ovs ctl interfaces, and the introduce of factory. 
I would suggest focus on what we have today given that we have assumed single bridge for most interfaces. We could refactor the current OVS interfaces when multiple bridges are needed, by which time I think the non-receiver function ListBridges and ofctlClientFactory may not be needed.

Reply:
Got it, then I would like to remove these changes.